tests/setup_test.go

```diff
@@ -176,7 +177,7 @@ func startAgola(ctx context.Context, t *testing.T, log zerolog.Logger, dir strin
 	return errCh, nil
 }
 
-func setup(ctx context.Context, t *testing.T, dir string, gitea bool) (*testutil.TestGitea, *config.Config) {
+func setup(ctx context.Context, t *testing.T, dir string, gitea bool, organizationMemberAddingMode config.OrganizationMemberAddingMode) (*testutil.TestGitea, *config.Config) {
```

Comment:
Instead of adding more options use the functional options pattern: https://commandcenter.blogspot.com/2014/01/self-referential-functions-and-design.html

When not passed the empty config default will be used.

Reply:
@sgotti I must open a new PR for this change?